packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/index.tsx

```diff
@@ -43,6 +44,9 @@ export const RainbowKitCustomConnectButton = () => {
 
               return (
                 <>
+                  <div className="flex mr-2">
+                    <BatchMemberStatuses></BatchMemberStatuses>
```

Comment:
Nitpick: If a component doesn't have any children, it's better to use a self-closing tag:

```jsx
<BatchMemberStatuses/>
```

Reply:
It makes sense, I'm new to react developmen and didn't know this was the standard, thanks for the feedback.